server/bundler/css-hot-reload.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Module dependencies
+ */
+var os = require( 'os' ),
+	fs = require( 'fs' ),
+	path = require( 'path' ),
+	crypto = require( 'crypto' ),
+	spawn = require( 'child_process' ).spawn,
+	debug = require( 'debug' )( 'css-hot-reload' ),
+	chalk = require( 'chalk' ),
+	chokidar = require('chokidar');
+
+/**
+ * Initialize server CSS hot-reloading logic
+ */
+function setup( io ) {
+
+	const SCSS_PATHS = [ './assets/stylesheets/', './client/' ];
```

Comment:
Yes definitely, I've tested it only under Linux Chrome/Firefox. I've planned a round of tests under Windows next.